File: app/src/main/java/com/example/josycom/flowoverstack/model/Question.java
package com.example.josycom.flowoverstack.model;

import com.google.gson.annotations.SerializedName;

import java.util.List;

public class Question {

    @SerializedName("tags")
    private List<String> tags = null;
    @SerializedName("body")
    private String body;
    @SerializedName("owner")
    private Owner owner;
    @SerializedName("is_answered")
    private Boolean isAnswered;
    @SerializedName("view_count")
    private Integer viewCount;
    @SerializedName("answer_count")
    private Integer answerCount;
    @SerializedName("score")
    private Integer score;
    @SerializedName("last_activity_date")
    private Integer lastActivityDate;
    @SerializedName("creation_date")
    private Integer creationDate;
    @SerializedName("question_id")
    private Integer questionId;
    @SerializedName("link")
    private String link;
    @SerializedName("title")
    private String title;

    public List<String> getTags() {
        return tags;
    }

    public void setTags(List<String> tags) {
        this.tags = tags;
    }

    public Owner getOwner() {
        return owner;
    }

    public void setOwner(Owner owner) {
        this.owner = owner;
    }

    public Boolean getIsAnswered() {
        return isAnswered;
    }

    public void setIsAnswered(Boolean isAnswered) {
        this.isAnswered = isAnswered;
    }

    public Integer getViewCount() {
        return viewCount;
    }

    public void setViewCount(Integer viewCount) {
        this.viewCount = viewCount;
    }

    public Integer getAnswerCount() {
        return answerCount;
    }

    public void setAnswerCount(Integer answerCount) {
        this.answerCount = answerCount;
    }

    public Integer getScore() {
        return score;
    }

    public void setScore(Integer score) {
        this.score = score;
    }

    public Integer getLastActivityDate() {
        return lastActivityDate;
    }

    public void setLastActivityDate(Integer lastActivityDate) {
        this.lastActivityDate = lastActivityDate;
    }

    public Integer getCreationDate() {
        return creationDate;
    }

    public void setCreationDate(Integer creationDate) {
        this.creationDate = creationDate;
    }

    public Integer getQuestionId() {
        return questionId;
    }

    public void setQuestionId(Integer questionId) {
        this.questionId = questionId;
    }

    public String getLink() {
        return link;
    }

    public void setLink(String link) {
        this.link = link;
    }

    public String getTitle() {
        return title;
    }

    public void setTitle(String title) {
        this.title = title;
    }

    public String getBody() {
        return body;
    }
}

File: app/src/main/java/com/josycom/mayorjay/flowoverstack/model/Question.kt
package com.josycom.mayorjay.flowoverstack.model

import com.google.gson.annotations.SerializedName

class Question {
    @SerializedName("tags")
    var tags: List<String>? = null

    @SerializedName("body")
    var body: String? = null

    @SerializedName("owner")
    var owner: Owner? = null

    @SerializedName("is_answered")
    var isAnswered: Boolean? = null

    @SerializedName("view_count")
    var viewCount: Int? = null

    @SerializedName("accepted_answer_id")
    var acceptedAnswerId: Int? = null

    @SerializedName("answer_count")
    var answerCount: Int? = null

    @SerializedName("score")
    var score: Int? = null

    @SerializedName("last_activity_date")
    var lastActivityDate: Int? = null

    @SerializedName("creation_date")
    var creationDate: Int? = null

    @SerializedName("last_edit_date")
    var lastEditDate: Int? = null

    @SerializedName("question_id")
    var questionId: Int? = null

    @SerializedName("link")
    var link: String? = null

    @SerializedName("title")
    var title: String? = null
}